integration_test/upgrade_test.go

```diff
@@ -14,4 +14,50 @@
 
 package integrationtest
 
-// TODO(ahmetb): implement upgrade tests (https://krew.dev/issues/233)
+import (
+	"crypto/sha256"
+	"io"
+	"os"
+	"path/filepath"
+	"testing"
+)
+
+func TestKrewUpgrade(t *testing.T) {
+	skipShort(t)
+
+	test, cleanup := NewTest(t)
+	defer cleanup()
+
+	cwd, err := os.Getwd()
```

Comment:
you don't need CWD, you can just do `filepath.Join("testdata","konfig.yaml")` –although I’m not sure if `go test` sets any CWD. If we do this:


    cd hack
    run-integration-tests.sh

this might not work.

Reply:
Indeed, `cwd` is obsolete here. I checked that it works when running the tests from different dirs as well.